test/support/factories/keyword_factory.ex

```diff
@@ -15,6 +15,18 @@ defmodule GoogleScraping.KeywordFactory do
         }
       end
 
+      def keyword_url_factory do
+        user = insert(:user)
+        keyword = insert(:keyword, user_id: user.id)
+
+        %KeywordUrl{
+          url: Faker.Lorem.word(),
+          user_id: user.id,
```

Comment:
Does t work?

```suggestion
          user: insert(:user),
```

Reply:
We discussed it before, there is something not set up for the factory, and we can pass here `user_id` only, not the `user`. Last time we decided that it was ok.